python/examples/4-profiling.py

```diff
@@ -134,31 +134,36 @@ def forward(
 #     </details>
 #
 
-
 # %%
 #
-# If you are trying to profile your own model, you can start here and create a
+# If you are trying to profile your own model, you can start here and create
 # ``MetatomicCalculator`` with your own model.
 
+# %%
+#
+# Profiling energy calculation
+# ----------------------------
+#
+# We will start with an energy-only calculator, which can be enabled with
+# ``do_energy_gradients_with_energy=False``.
 
-atoms.calc = MetatomicCalculator("exported-model.pt")
+atoms.calc = MetatomicCalculator(
+    "exported-model.pt",
+    do_energy_gradients_with_energy=False,
+)
 
 # %%
 #
 # Before trying to profile the code, it is a good idea to run it a couple of times to
 # allow torch to warmup internally.
 
-atoms.get_forces()
-atoms.get_potential_energy()
+for _ in range(10):
+    atoms.get_potential_energy()
```

Comment:
Do we not cache the energy on the `atoms` object making this a no-op after the first calculation?